api/src/test/java/com/epam/pipeline/manager/pipeline/PipelineRunManagerLaunchTest.java

```diff
@@ -368,6 +409,13 @@ private void mock(final Tool tool) {
         doReturn(tool).when(toolManager).resolveSymlinks(eq(IMAGE));
     }
 
+    private Map<Long, List<RunStatus>> getStatusMap() {
+        final Map<Long, List<RunStatus>> map = new HashMap<>();
+        map.put(ID, Arrays.asList(TEST_STATUS_1, TEST_STATUS_2));
+        map.put(ID_2, singletonList(TEST_STATUS_3));
```

Comment:
It feels odd to have static import for `Collections.singletonList` and to don't have it for `Arrays.asList`. I would recommend using either one syntax or another not both.

```java
# bad
map.put(ID, Arrays.asList(TEST_STATUS_1, TEST_STATUS_2));
map.put(ID_2, singletonList(TEST_STATUS_3));

# good
map.put(ID, Arrays.asList(TEST_STATUS_1, TEST_STATUS_2));
map.put(ID_2, Collections.singletonList(TEST_STATUS_3));

# good too
map.put(ID, asList(TEST_STATUS_1, TEST_STATUS_2));
map.put(ID_2, singletonList(TEST_STATUS_3));
```

Reply:
Done!